docs/changelog.rst

```diff
@@ -5,6 +5,12 @@ CHANGELOG
 2.87.2+dev (XXXX-XX-XX)
 -----------------------
 
+
+**Minor improvements**
+- Set begin_date, end_date not null for touristic event
+- Rename metting_time to start_time
```

Comment:
- metting_time > meeting_time
- You should mention in which module (Touristic events)